h/templates/admin/groups.html.jinja2

```diff
@@ -8,6 +8,12 @@
     On this page you can see a list of all the groups and their details.
   </p>
 
+  <p>
+    <a href="/admin/groups?fmt=csv">
```

Comment:
This URL should probably be generated using `request.route_url`. It would probably look something like `request.route_url('admin_groups_csv')``` if you make the changes suggested earlier in this PR.